test_deps.ts

```diff
@@ -1,3 +1,2 @@
 // Add your test dependencies in here
-export { test, runTests } from "https://deno.land/std@v0.32.0/testing/mod.ts";
 export { assertEquals } from "https://deno.land/std@v0.32.0/testing/asserts.ts";
```

Comment:
Shouldn't this version be updated too? 

```suggestion
export { assertEquals } from "https://deno.land/std@v0.83.0/testing/asserts.ts";
```

Reply:
Oops! I pushed a new commit for this one, thank you.